pkg/cmd/server/kubernetes/master.go

```diff
@@ -83,6 +82,12 @@ func (c *MasterConfig) InstallAPI(container *restful.Container) []string {
 	}
 }
 
+func (c *MasterConfig) RunNamespaceController() {
+	namespaceController := namespace.NewNamespaceManager(c.KubeClient)
+	namespaceController.Run(1 * time.Minute)
```

Comment:
This seems slow.  Does this mean that my namespace may not be cleaned up for a minute?  During that minute, can I keep creating things?